layouts/partials/head.html

```diff
@@ -86,7 +86,7 @@
 {{ end }}
 <meta property="og:type" content="{{ $ogType }}">
 <script src="{{ "js/jquery-ui-1.12.1.min.js" | relURL }}"></script>
-<script src="{{ "js/sweetalert-2.1.2.min.js" | relURL }}"></script>
+<script async src="{{ "js/sweetalert-2.1.2.min.js" | relURL }}"></script>
```

Comment:
@sftim , The script is added to the "head" partial as well before the helper function?
Is it needed in both places?
👍  to cleaning up footer-scripts.html.

Reply:
Oops, I'll cut it out of the head partial.